cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -1,12 +1,12 @@
 [id="understanding-odo"]
 = Understanding OpenShift Do
-include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/developer-cli-odo-attributes.adoc[]
 include::modules/common-attributes.adoc[]
 :context: understanding-odo
 toc::[]
 
-{odo-title-full} (`{odo-title}`) is a fast and easy-to-use CLI tool for creating applications on {product-title}. `{odo-title}` allows developers to concentrate on creating applications without the need to administrate an {product-title} cluster itself.
-Creating deployment configurations, build configurations, service routes and other {product-title} elements are all automated by `{odo-title}`.
+`{odo-title}` is a fast and easy-to-use CLI tool for creating applications on {product-title} and Kubernetes. `{odo-title}` allows developers to concentrate on creating applications without the need to administrate a cluster itself.
```

Comment:
s/easy-to-use/easy to use 
I think this sounds like marketing speak, so I suggest removing "fast and easy-to-use" altogether.

s/administrate/administer